### PR TITLE
Kang-Ding-Xin-NYCU proposal for PolygonAreaCalculator

### PR DESCRIPTION
**Pull request by Kang-Ding-Xin-NYCU**
---
A good PR should include the following parts:

- [x] Provide a clear title
- [x] Provide meaningful descriptions in the PR and the commit logs
- [x] Name your branch like ``<github-user-name>-hw#``, e.g., ``myname-hw2-attempt1``
- [x] Merge from your specific branch (***not master***) to the `hw*` branch (***not master***, again)
- [x] Exclude irrelevant files, such as binaries, text files, or dot files
- [x] Make sure tests/CI pass

### CODE REVIEW

**Review by yungyuc — CHANGES_REQUESTED** [review]
You wrongly deleted `project/README.rst`.  Please fix it before proceeding.

Your commit history should not contain the wrong removal of the file.  When fixing the wrong deletion, please squash too.

When requesting for a next review, please leave a global comment summarizing what you change.

**Comment by Kang-Ding-Xin-NYCU** [comment]
Use git reset to reset the branch and copy the file that I already wrote.
Finally, git push -f.

File: project/Kang-Ding-Xin-NYCU/README.rst [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+* result : the area of polygons and the image of polygons
+
+API Description
+===============
+code
+
+.. code:: c++
+
+    class polygon{
+        vector<vertices> verticeslist;
```
Comment (yungyuc) [inline project/Kang-Ding-Xin-NYCU/README.rst]: Please specify the value data type of the coordinates.

File: project/Kang-Ding-Xin-NYCU/README.rst [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+Engineering Infrastructure
+==========================
+
+* build : make
+* version control : git
+
+Schedule
+========
+
+* Week 1 (09/20):survey the method to implement the area calculation and drawing image
```
Comment (yungyuc) [inline project/Kang-Ding-Xin-NYCU/README.rst:49]: Please include automatic testing and CI early in the schedule.  Think about how it improves your development productivity.

File: project/Kang-Ding-Xin-NYCU/README.rst [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+PolygonAreaCalaulator
+
+`Kang-Ding-Xin-NYCU <https://github.com/Kang-Ding-Xin-NYCU>`__ for
+`PolygonAreaCalaulator <https://github.com/Kang-Ding-Xin-NYCU/PolygonAreaCaculator/README.rst>`__
+:https://github.com/Kang-Ding-Xin-NYCU/PolygonAreaCaculator
+
+Problem to Solve
+================
+
+* Easily get the the area of polygons and the image of polygons
```
Comment (yungyuc) [inline project/Kang-Ding-Xin-NYCU/README.rst]: Please specify how the code handles holes in polygons and nested holes.  (There are basically two styles to represent holes: cutlines and islands.)

**Review by yungyuc — CHANGES_REQUESTED** [review]
- [x] Please add an entry in `project/README.rst` pointing to your proposal.
- [x] Specify the value data type of coordinates.
- [x] Include automatic testing and CI.
- [x] Specify how to handle holes.

**Comment by yungyuc** [comment]
@Kang-Ding-Xin-NYCU please fix all issues and update as soon as possible.  I will stop accepting proposals any time.  It's long overdue.

**Comment by Kang-Ding-Xin-NYCU** [comment]
- [x] Please add an entry in project/README.rst pointing to your proposal.
- [x] Specify the value data type of coordinates.
- [x] Include automatic testing and CI.
- [x] Specify how to handle holes.

**Comment by yungyuc** [comment]
> * [ ]  Please add an entry in project/README.rst pointing to your proposal.

You have not done it.

**Comment by yungyuc** [comment]
Please review merged PRs: https://github.com/yungyuc/nsdhw_24au/pulls?q=is%3Apr+label%3Aproject+is%3Aclosed

**Comment by Kang-Ding-Xin-NYCU** [comment]
I have changed the link but I am not very sure that is right

**Comment by yungyuc** [comment]
> I have changed the link but I am not very sure that is right

You have not got it correctly.  You need to add an entry in the file `project/README.rst`, not in your proposal file.  Please review the merged PRs.

@jteng2127, if you have time, could you please help @Kang-Ding-Xin-NYCU for his PR?

**Comment by Kang-Ding-Xin-NYCU** [comment]
Oh, I got it. I have add an entry in project/README.rst.

**Comment by jteng2127** [comment]
Yes, I could help

If there is still any issue, @Kang-Ding-Xin-NYCU when are you available?

**Review by yungyuc — APPROVED** [review]
Plain text should use less than 80 characters in a line.  Long lines are hard to review and comment.

The proposal is OK.  Please start prototyping.

**Comment by jteng2127** [comment]
@yungyuc I think the entry in `project/README.rst` is misplaced

image

**Comment by yungyuc** [comment]
> @yungyuc I think the entry in `project/README.rst` is misplaced
> 
> image

You are right.  But it's faster for me to change it myself: https://github.com/yungyuc/nsdhw_24au/commit/ed6f48bf0412b8a5f9324cd055c49a4906700e8c

**Pull request merged**